extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1046,6 +1054,7 @@ export class AmpA4A extends AMP.BaseElement {
    * @private
    */
   renderNonAmpCreative_() {
+    this.promiseErrorHandler_(new Error('fallback to 3p'));
```

Comment:
This is not actually an error as we expect non AMP creatives.  Are we sure we want to report it as such?